torch/distributed/tensor/parallel/_utils.py

```diff
@@ -149,4 +149,6 @@ def _create_1d_device_mesh(device_mesh: DeviceMesh, tp_mesh_dim: int = 0) -> Dev
     dim_mesh_1d = pg_ranks_by_dim[torch.any(pg_ranks_by_dim == cur_rank, 1), :]
 
     sub_pg = device_mesh.get_dim_groups()[tp_mesh_dim]
-    return DeviceMesh(device_mesh.device_type, dim_mesh_1d.squeeze(), [sub_pg])
+    mesh_1d = DeviceMesh(device_mesh.device_type, dim_mesh_1d.squeeze(), init_process_groups=False)
+    mesh_1d._dim_groups = [sub_pg]
```

Comment:
Can we make override an API?

Reply:
Yeah I would prefer to do so if we really need this, but currently I think we should figure out how to slice a 2-D mesh to a 1-D mesh, if override is needed for the slice, we can definitely expose an API to do it